### ISSUE TITLE
testFramework is undefined in setup() when using ava.

### ISSUE THREAD

**Issue by vrecluse** [wallabyjs/public#1078]
### Issue description or question
I attempted to pass --serial flag to ava but found no where in the document mentioned how to pass.
Tried to use setup() in config file, but the testFramework is undefined.

### Wallaby.js configuration file

```javascript
module.exports = function () {
  process.env.NODE_ENV = 'test'

  return {
    files: [
      'config/**/*.json',
      'src/**/*.ts',
      '!src/**/*.spec.ts',
    ],

    tests: [
      'src/**/*.spec.ts',
    ],

    env: {
      type: 'node'
    },

    testFramework: 'ava',

    setup: function (wallaby) {
      var ava = wallaby.testFramework
      console.log(ava) // prints undefined here.
    },

    debug: true,
  }
}
```

### Code editor or IDE name and version

Visual Studio Code v1.x

### OS name and version

OSX

**Comment by ArtemGovorov** [comment]
It is not exposed by design, we didn't find that one would need to access/configure.  Regarding the `--serial` flag, you don't need to pass it, because wallaby is using it by default with `ava` and has its own mechanism of parallelising tests.

**Comment by vrecluse** [comment]
And is there any way to turn off the parallel mechanism of wallaby.js?

**Comment by ArtemGovorov** [comment]
@vrecluse Yes, you may use the [`workers` setting](https://wallabyjs.com/docs/config/workers.html) for it:

```javascript
 module.exports = function () {
   return {
     ...
     workers: {
       initial: 1,
       regular: 1
     }
   };
 };
```

**Comment by smcenlly** [comment]
We have updated Wallaby to allow you to set test runner options for Ava. The testFramework for Ava now provides an empty object { } (was previously undefined). You may set ava's options by setting the properties of the testFramework. This is available in Wallaby core v1.0.768+.

Please note that the following properties will always be overridden by Wallaby: file, projectDir, updateSnapshots.
